core/scss/utilities/mixins/link/_more-link.scss

```
@@ -1,21 +1,24 @@
 @charset "UTF-8";
 
 //
-// @more-link($more-link, $more-link-hover)
+// @more-link($color, $hover)
 //
 // Default more link styling, with raquo.
 //
+// $color - Default up color
+// $hover - Default hover color
+//
 // Style guide: Mixins.Link.more-link
 //
-@mixin more-link($more-link, $more-link-hover) {
+@mixin more-link($color: $color-bright-red, $hover: $color-black) {
   &::after {
-    color: $color-bright-red;
+    color: $color;
     content: '\00BB';
     display: inline-block;
     margin-left: 4px;
```

See previous related comment on margin mixin. This could be:

```suggestion
    @include margin(null null null 4px);
```